test/hack/test-ltp.yml

```diff
@@ -16,11 +16,7 @@ onboot:
     capabilities:
      - all
   - name: poweroff
-    image: "linuxkit/poweroff:961412b8ef5c5285de0d40ec076701d955eaa084"
-    pid: host
-    capabilities:
-      - CAP_SYS_BOOT
-    readonly: true
+    image: "linuxkit/poweroff:5673236900c6beae35bbceec90ded4b9add3f5ae"
```

Comment:
oh yes I did but I shouldnt have, going to revert. It makes little sense having `command` in yaml as it is already in image